libs/monitor.js

```diff
@@ -55,6 +56,12 @@ function initRoutes(cb) {
 
   //app routes
   app.get(/\/$|\/admin$/, require('../server/routes/index'));
+  app.use("/box/srv/1.1/admin/authpolicy/auth", require("../server/routes/auth"));
+  
+  if(process.env.AUTH_POLICY){
+    //app.use(mbaasExpress.fhauth({ cache: true, expire: 60 * 60 }));
```

Comment:
ah yes, well as discussed with John the rest of the api calls to the server would have to send along the sessionToken. $fh.auth was designed with the expectation that you would be using $fh.cloud which sends along the sessionToken. $fh was not used at all before I added this so there is some work there. Until that is done I commented out the server side check.